doc/api/resources/waitinglist.rst

```diff
@@ -13,7 +13,9 @@ Field                                 Type                       Description
 ===================================== ========================== =======================================================
 id                                    integer                    Internal ID of the waiting list entry
 created                               datetime                   Creation date of the waiting list entry
+name                                  string                     Name of the user on the waiting list
```

Comment:
* Probably add "or ``null``" both here and for the phone field so API users know these fields are nullable
* This is missing name_parts (see orders.rst for an example)